requirements.txt

```diff
@@ -9,3 +8,4 @@ requests
 six
 unittest-xml-reporting
 ibex_bluesky_core
+genie_python @ git+https://github.com/IsisComputingGroup/genie@a4a1f2f1468107a6a20d71a82caa6acbdcc4fa8e
```

Comment:
When merging, merge the corresponding `genie` change first, then edit this to point at main, then merge this pr